package/batocera/core/batocera-configgen/configgen/configgen/generators/libretro/libretroMAMEConfig.py

```diff
@@ -454,9 +454,18 @@ def generateMAMEConfigs(playersControllers: ControllerMapping, system: Emulator,
 
     # Write command line file
     cmdFilename = cmdPath / f"{romDrivername}.cmd"
-    cmdFile = cmdFilename.open("w")
-    cmdFile.write(' '.join(str(item) for item in commandLine))
-    cmdFile.close()
+    # Check to see whether user provided a custom cmd file, at either a default location, or specified in batocera.conf
+    defaultCustomCmdFilepath = f'{rom}.cmd'
```

Comment:
If you do `rom.with_suffix(f'{rom.suffix}.cmd')`, you'll get a path object and won't need to use `Path()` on the next line

Reply:
The filename variable is used again in line 460 when copying. If we use that, we'll have to construct the filename again for line 460, no?

Reply:
Ah, my bad. I see your point now. Will fix.

Reply:
lol. too late. I have other stuff to push and will include this.